### PR TITLE
TELCODOCS-1120 removing section on incompatibility with IP failover

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1120]: Misleading text in MetalLB incompatibility section
[OCPBUGS-9222]: This section has generated quite a lot of discussions on e-mails and on slack
Version(s):
main, 4.13, 4.12, 4.11, 4.10, 4.9

Issue:
https://issues.redhat.com/browse/TELCODOCS-1120

Link to docs preview:https://57579--docspreview.netlify.app/openshift-enterprise/latest/networking/metallb/about-metallb.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
@asood-rh can you please review this PR providing clarity around MetalLB and IP failover

File: modules/nw-metallb-when-metallb.adoc
Comment (stevsmit) [inline modules/nw-metallb-when-metallb.adoc:15]: the virtual router redundancy protocol? (with "the" added)?

File: modules/nw-metallb-when-metallb.adoc
Comment (stevsmit) [inline modules/nw-metallb-when-metallb.adoc:17]: consider: 

s/ "more preferable" / "preferable" or "is the preferring option"

**Comment by stevsmit** [comment]
@kquinn1204 2 nits. Be sure to squash your commits as well. Thanks!

**Review by kelbrown20 — COMMENTED** [review]
@kquinn1204 Hey Kevin! Just want to confirm that this has gone through QE review before merging? I did not see the qe approval marked or a comment, but I may have missed it. Otherwise this LGTM as well.

**Comment by kquinn1204** [comment]
@asood-rh I need QE approval to get this change implemented it is a clarification on MetalLB and IP failover in docs that causes some confusion. Can you set a LGTM please?

**Comment by kquinn1204** [comment]
@kelbrown20 I got a LGTM in keeping with the process. Thanks for flagging, so should be good to merge now.

**Comment by kquinn1204** [comment]
@kelbrown20 I got a LGTM in keeping with the process. Thanks for flagging, so should be good to merge now.

**Comment by kquinn1204** [comment]
@kelbrown20 I got a LGTM in keeping with the process. Thanks for flagging, so should be good to merge now.

**Comment by kquinn1204** [comment]
@kelbrown20 I got a LGTM in keeping with the process. Thanks for flagging, so should be good to merge now.

**Review by russellb — CHANGES_REQUESTED** [review]
@fedepaol can you give this another look?

File: modules/nw-metallb-when-metallb.adoc
Comment (fedepaol) [reply modules/nw-metallb-when-metallb.adoc:15]: :+1 and thanks for the review!

**Comment by kelbrown20** [comment]
@kquinn1204 Looks like the CP to 4.9 failed so this need a manual one :'( . I can merge the manual CP when ready

**Pull request merged**
